cpp/bench/CMakeLists.txt

```diff
@@ -27,11 +27,10 @@ if(BUILD_CUML_BENCH)
     sg/kmeans.cu
     sg/main.cpp
     sg/rf_classifier.cu
-    sg/rf_regressor.cu
+    # sg/rf_regressor.cu
     sg/svc.cu
-    sg/svr.cu
-    sg/umap.cu
-    )
+    sg/svr.cu)
+    # sg/umap.cu)
```

Comment:
what is umap waiting for? https://github.com/rapidsai/raft/pull/46 has been merged in the raft side and this PR seems to change to point to the latest commit there indeed

Reply:
Done.